module/gridss.nf

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env nextflow
+
+log.info """\
+------------------------------------
+             G R I D S S 2
+------------------------------------
+Docker Images:
+- docker_image_gridss: ${params.docker_image_gridss}
+"""
+
+include { generate_standard_filename; sanitize_string } from '../external/pipeline-Nextflow-module/modules/common/generate_standardized_filename/main.nf'
+
+process preprocess_BAM_GRIDSS {
+    container params.docker_image_gridss
+
+    publishDir "${params.workflow_output_dir}/intermediate",
+        pattern: "${bam_name}.gridss.working/*",
+        mode: "copy",
+        saveAs: {
+            "${output_filename}.${sanitize_string(file(it).getName().replace("${bam_name}.", ""))}"
+            }
+
+    publishDir "${params.log_output_dir}/process-log",
+        pattern: ".command.*",
+        mode: "copy",
+        saveAs: { "${task.process.replace(':', '/')}/log${file(it).getName()}" }
```

Comment:
**comment (blocking)**: These log files are being over-written since this process runs more than once